src/store-tree/index.ls

```diff
@@ -21,10 +21,7 @@ class StoreTree extends StoreNode
 
 
   $get-error: ->
-    for own key of @_children
-      err = @[key].$get-error!
-      return err if err?
-    null
+    @_children |> values |> find (.$get-error!) |> -> it?.$get-error! or null
```

Comment:
Okay. What about:

```ls
@_children |> find (.$get-error!?.get-error!) 
```

It does return `undefined` instead of `null`. But referring to the conversation about falsy below, code should not put too much emphasis on the difference between these two values. The semantic difference between them is that `undefined` means "this variable was never assigned a value", while `null` means this variable was given a value, and the value is unknown at this point. For most applications, including this one, this distinction is a moot point, and both values should be treated the same. If `null` was the billion dollar mistake, `undefined` is the same mistake done twice in the same language. The existential operator (`?`) treats them the same as well.